lib/openshift/machine_config_pool.rb

```diff
@@ -8,5 +8,10 @@ def spec(user: nil, cached: true, quiet: false)
       rr = raw_resource(user: user, cached: cached, quiet: quiet).dig("spec")
       MachineConfigPoolSpec.new rr
     end
+
+    def machineconfig(mc_name, user: nil, cached: true, quiet: false)
```

Comment:
@jhou1 since this is a `status` property, I'm wondering if we should just default to `cached: false`...that way you don't need to specify that when calling it.  WDYT @akostadinov ?  LGTM otherwise.

Reply:
We should always default to `cached: true` because most common use cases benefit with performance from this. When we loop-wait people should get used to doing `cached: false`. It is simpler convention than trying to think what would be more common for each individual method and then never know what is the default.